libnmstate/nm/ovs.py

```diff
@@ -184,46 +185,50 @@ def _get_patch_setting(act_con):
     return None
 
 
-def get_port(nm_device):
-    return nm_device.get_slaves()
+def _get_lag_nmstate_port_info(nm_dev_ovs_port):
+    port_info = {}
 
+    OVS_LAG = OB.Port.LinkAggregation
```

Comment:
Is a function modifying a constant?

Reply:
Just create alias for the long constant.